### PR TITLE
PWN-6505 - Sentry optimization

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6505

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt [hunk 1]
```diff
@@ -57,7 +57,7 @@ class UsernameInteractor(
             }
             crashLogger.setCustomKey("username", usernameDetails?.username?.fullUsername.orEmpty())
         } catch (error: Throwable) {
-            Timber.i(error, "Failed to restore username for ${owner.base58Value}")
+            Timber.e(error, "Failed to restore username for ${owner.base58Value}")
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt:60]: remove `owner.base58Value`

File: app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SeedPhraseProvider.kt [hunk 2]
```diff
@@ -30,7 +30,7 @@ class SeedPhraseProvider(
         set(value) {
             field = value
             saveSeedPhraseToStorage(value)
-            Timber.tag(TAG).i("updating user seed phrase: $value")
+            Timber.tag(TAG).i("updating user seed phrase: ${value.size}")
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SeedPhraseProvider.kt:33]: remove `value.size`

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt:60]: why? we need to know which user is failed to restore username

File: app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SeedPhraseProvider.kt (hunk 2 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SeedPhraseProvider.kt]: replaced with isEmpty to understand are we removing seedphrase from shared prefs

on reset we are doing seedPhrase = emptyList()

File: app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt (hunk 1 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt:60]: someone can easily pass an owner as a secret key to this function. 
we will know which user is failed from anywhere

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/auth/interactor/UsernameInteractor.kt:60]: ok, removing

**Review by 1sakov — APPROVED** [review]

**Pull request merged**